source/faq/questions/using-cypress-faq.md

```diff
@@ -147,7 +147,10 @@ When you load your application using `cy.visit()`, Cypress will wait for the `lo
 
 **_In CI, how do I make sure my server has started?_**
 
-There are a couple really great modules that we recommend using for this, {% url '`wait-on`' https://www.npmjs.com/package/wait-on %} and {% url '`start-server-and-test`' https://github.com/bahmutov/start-server-and-test %}.
+We recommend these great modules for this use case:
+
+* {% url '`wait-on`' https://www.npmjs.com/package/wait-on %}
+* {% url '`start-server-and-test`' https://github.com/bahmutov/start-server-and-test %}
```

Comment:
Reworded this because the original had a grammatical error and lists of hyperlinked modules "pops" better for me.

> I appreciate that last point is subjective 🤗